torchok/constructor/logger.py

```diff
@@ -35,6 +40,43 @@ def create_outputs_path(log_dir: str, experiment_name: str, timestamp: str = Non
     return full_outputs_path
 
 
+def _flatten_dict(
+    params: MutableMapping[Any, Any], delimiter: str = "/", parent_key: str = ""
+) -> Dict[str, Any]:
+    """Flatten hierarchical dict, e.g. ``{'a': {'b': 'c'}} -> {'a/b': 'c'}``.
+    Args:
+        params: Dictionary containing the hyperparameters
+        delimiter: Delimiter to express the hierarchy. Defaults to ``'/'``.
+    Returns:
+        Flattened dict.
+    Examples:
+        >>> _flatten_dict({'a': {'b': 'c'}})
+        {'a/b': 'c'}
+        >>> _flatten_dict({'a': {'b': 123}})
+        {'a/b': 123}
+        >>> _flatten_dict({5: {'a': 123}})
```

Comment:
The examples look in the same fashion. Let's leave the first one and add a case with a list